project.inlang.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema":"https://inlang.com/schema/project-settings",
+    "sourceLanguageTag": "en",
+    "languageTags": ["ar-SA", "ca-ES", "de-DE", "en", "es-VE", "eu-ES", "fa", "fr-CA", "fr-FR", "hu-HU", "id-ID", "it-IT", "ja-JP", "ko-KR", "pl-PL", "pt-BR", "pt-PT", "ru-RU", "sv", "tr-TR", "zh-CN", "zh-TW"],
```

Comment:
Hi @jannesblobel. Are they auto-generated? 

Should we update here if we need to support a new language?

Reply:
@LucasXu0  good questions 

> Are they auto-generated?

Our migrate function generates these language tags.

> Should we update here if we need to support a new language?

Its dependence on who adds a new language: 
1. you add a new language via the inlang editor -> the Editor will add this new language tag for you/the user 

2. a developer manually adds a new file. He must add the language tag to this array. This approach is because sometimes a developer adds a new language but does not want it to be available in the editor.

Reply:
@jannesblobel So, these tags are intended for the InLang editor? May I ask why you don't simply retrieve the languages from the languages folder?